packages/gestalt-datepicker/package.json

```diff
@@ -27,20 +27,6 @@
     "prepublish": "rollup -c rollup.config.js",
     "watch": "rollup -c rollup.config.js --watch"
   },
-  "devDependencies": {
```

Comment:
Moved these up - otherwise we would have to update them in 2 locations each time + we might forget 1 location.